sympy/printing/lambdarepr.py

```diff
@@ -118,6 +118,31 @@ def _print_Function(self, e):
                                 func_name)
         return "%s(%s)" % (nstr, self._print_seq(e.args))
 
+    def _print_Piecewise(self, expr):
+        "Piecewise function printer"
+        exprs = [self._print(arg.expr) for arg in expr.args]
+        conds = [self._print(arg.cond) for arg in expr.args]
+        # If [default_value, True] is a (expr, cond) sequence in a Piecewise object
+        #     it will behave the same as passing the 'default' kwarg to select()
+        #     *as long as* it is the last element in expr.args.
+        # If this is not the case, it may be triggered prematurely.
+        ans = []
+        parenthesis_count = 0
+        is_last_cond_True = False
+        for cond, expr in zip(conds, exprs):
+            if cond == 'True':
+                ans.append(expr)
+                is_last_cond_True = True
+                break
+            else:
+                ans.append('where(%s,%s,' % (cond, expr))
```

Comment:
This may be need some changes to add a spacing between the comma.
It now prints as `"evaluate('where(b,a,where(d,c,e))', truediv=True)"`

Reply:
I will work on it.